src/test/test-seccomp.c

```diff
@@ -1008,16 +1008,9 @@ static int real_open(const char *path, int flags, mode_t mode) {
 }
 
 static int try_fchmodat2(int dirfd, const char *path, mode_t mode, int flags) {
```

Comment:
no point in retaining this static wrapper anymore

Reply:
I feel like the `try_fchmodat2` name would be more descriptive in the context of the test, but a comment where the new fchmodat wrapper is used will do.